.github/workflows/ci.yml

```diff
@@ -17,13 +17,13 @@ jobs:
       - name: Setup latest deno version
         uses: denolib/setup-deno@v2
         with:
-          deno-version: v1.3.3
+          deno-version: v1.4.0
 
       - name: Format
         run: deno fmt --check
 
       - name: Lint
-        run: deno lint --unstable
+        run: deno lint --unstable --ignore=recipes/react-template/src/
```

Comment:
This is for the CSS imports, right? I'm thinking whether this would be the correct approach... Maybe we should add lint ignore strings instead because that would _probably_ be more handy for people re-using the recipes? What do you think @lucacasonato ?

Reply:
yup - that is what I did now

Reply:
Ah it's for the no-undef rule.